src/main/game/ControlFileReader.java

```diff
@@ -0,0 +1,42 @@
+package game;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import javafx.scene.input.KeyCode;
+
+public class ControlFileReader {
+	File controlsFile = new File("config/controls.cfg");
```

Comment:
I'll add the final String, but I don't see the point of initializing it in the constructor when it's functionally identical and less readable.